redis/Makefile

```diff
@@ -0,0 +1,98 @@
+
+REDIS_VERSION:=3.0.7
```

Comment:
Maybe you just can re-license the file under LPGL? If it's yours, that should be no problem.